### PR TITLE
Fix get all applications response

### PR DESCRIPTION
**Pull request by Krishanx92**

### CODE REVIEW

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIM678ApplicationCreationTestCase.java [hunk 1]
```diff
@@ -250,16 +252,14 @@ public void testApplicationCreationWithCustomAttributes(String applicationName,
...
 
     public void removeAllApps() throws Exception {
         //delete created applications
-        HttpResponse getAllAppResponse = restAPIStore.getAllApp();
-        String[] array = getAllAppResponse.getData().replace("[", "")
-                .replace("]", "").split(",");
-        for (int app = 0; (array.length - 1) > app; app++) {
-            restAPIStore.deleteApplication(array[app]);
+        ApplicationListDTO getAllAppResponse = restAPIStore.getAllApps();
+        List<ApplicationInfoDTO> array = getAllAppResponse.getList();
```
Comment (mushthaq33) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIM678ApplicationCreationTestCase.java]: Shall we rename this to applicationDTOList or something similar?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIM678ApplicationCreationTestCase.java [hunk 2]
```diff
@@ -250,16 +252,14 @@ public void testApplicationCreationWithCustomAttributes(String applicationName,
...
+        for (int app = 0; array.size() > app; app++) {
```
Comment (mushthaq33) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIM678ApplicationCreationTestCase.java]: Shall we use a foreach loop here?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIM678ApplicationCreationTestCase.java (hunk 1 above)
Comment (Krishanx92) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIM678ApplicationCreationTestCase.java]: Fixed

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIM678ApplicationCreationTestCase.java (hunk 2 above)
Comment (Krishanx92) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIM678ApplicationCreationTestCase.java]: Fixed

**Pull request merged**